src/commands/access-profile/AccessProfilePaginator.ts

```diff
@@ -0,0 +1,19 @@
+import { AccessProfile } from "../../models/AccessProfile";
+import { IdentityNowClient } from "../../services/IdentityNowClient";
+
+/**
+ * Don't actually need this class, so will remove it in future as the getAccessProfiles()
```

Comment:
The idea here is to "yield" results as they are coming.
So we write continuously the results, limiting the memory consumption.